examples/jsm/nodes/accessors/FatPointsMaterialNode.js

```diff
@@ -0,0 +1,21 @@
+import MaterialNode from './MaterialNode.js';
+import { addNodeClass } from '../core/Node.js';
+import { nodeImmutable } from '../shadernode/ShaderNode.js';
+
+class FatPointsMaterialNode extends MaterialNode {
```

Comment:
I don't think we need a separate node for this property, can we just add `pointwidth` to the main MaterialNode?

Reply:
@LeviPesin Thank you for your review. For each of your comments, can you please be more specific about the code to be removed/changed and the code to add? I am still trying to understand these concepts. :-)

Reply:
I will file a new PR for this after this one is merged.

Reply:
OK, resolving...